pkg/virt-handler/rest/console.go

```diff
@@ -0,0 +1,125 @@
+package rest
+
+import (
+	"github.com/emicklei/go-restful"
+	"github.com/gorilla/websocket"
+	"github.com/libvirt/libvirt-go"
+	"io"
```

Comment:
Put a space between the go standard ones like io and the github ones, and the tooling will sort them within the groups.  That does not have to be in this patch.